### ISSUE TITLE
"Check for Updates" doesn't work, GUI still fails to load even after manually updating by copypasting files from main page into installation

### ISSUE THREAD

**Issue by Zyberetro** [C0untFloyd/roop-unleashed#19]
**Describe the bug**
I've tried updating roop multiple times, every single time, it says it's up to date, same error about the face enhancer not being found, still doesn't load the GUI. I had the idea of copy-pasting all the files from the source code (the one on the main page with the fixes, not the newest release one, which is the old one). Here is the log from the command window:

Launching roop unleashed - please edit windows_run.bat to customize commandline arguments
Check for Updates? [y/n]y
HEAD is now at b42f939 Bugfix: frame processors
Already up to date.
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu118
Ignoring torch: markers 'sys_platform == "darwin"' don't match your environment
Ignoring torchvision: markers 'sys_platform == "darwin"' don't match your environment
Ignoring onnxruntime: markers 'sys_platform == "darwin" and platform_machine != "arm64"' don't match your environment
Ignoring onnxruntime-silicon: markers 'sys_platform == "darwin" and platform_machine == "arm64"' don't match your environment
Requirement already satisfied: numpy==1.23.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 3)) (1.23.5)
Requirement already satisfied: opencv-python==4.7.0.72 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 4)) (4.7.0.72)
Requirement already satisfied: onnx==1.14.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 5)) (1.14.0)
Requirement already satisfied: insightface==0.7.3 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 6)) (0.7.3)
Requirement already satisfied: psutil==5.9.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 7)) (5.9.5)
Requirement already satisfied: tk==0.1.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 8)) (0.1.0)
Requirement already satisfied: customtkinter==5.2.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 9)) (5.2.0)
Requirement already satisfied: pillow==9.5.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 10)) (9.5.0)
Requirement already satisfied: torch==2.0.1+cu118 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 11)) (2.0.1+cu118)
Requirement already satisfied: torchvision==0.15.2+cu118 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 13)) (0.15.2+cu118)
Requirement already satisfied: onnxruntime-gpu==1.15.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 17)) (1.15.0)
Requirement already satisfied: protobuf==4.23.2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 18)) (4.23.2)
Requirement already satisfied: tqdm==4.65.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 19)) (4.65.0)
Requirement already satisfied: codeformer-pip==0.0.4 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 20)) (0.0.4)
Requirement already satisfied: gfpgan==1.3.8 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from -r requirements.txt (line 21)) (1.3.8)
Requirement already satisfied: typing-extensions>=3.6.2.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from onnx==1.14.0->-r requirements.txt (line 5)) (4.7.1)
Requirement already satisfied: requests in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (2.31.0)
Requirement already satisfied: matplotlib in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (3.7.2)
Requirement already satisfied: scipy in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (1.11.1)
Requirement already satisfied: scikit-learn in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (1.3.0)
Requirement already satisfied: scikit-image in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (0.21.0)
Requirement already satisfied: easydict in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (1.10)
Requirement already satisfied: cython in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (0.29.36)
Requirement already satisfied: albumentations in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (1.3.1)
Requirement already satisfied: prettytable in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from insightface==0.7.3->-r requirements.txt (line 6)) (3.8.0)
Requirement already satisfied: darkdetect in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from customtkinter==5.2.0->-r requirements.txt (line 9)) (0.8.0)
Requirement already satisfied: filelock in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from torch==2.0.1+cu118->-r requirements.txt (line 11)) (3.12.2)
Requirement already satisfied: sympy in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from torch==2.0.1+cu118->-r requirements.txt (line 11)) (1.12)
Requirement already satisfied: networkx in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from torch==2.0.1+cu118->-r requirements.txt (line 11)) (3.1)
Requirement already satisfied: jinja2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from torch==2.0.1+cu118->-r requirements.txt (line 11)) (3.1.2)
Requirement already satisfied: coloredlogs in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from onnxruntime-gpu==1.15.0->-r requirements.txt (line 17)) (15.0.1)
Requirement already satisfied: flatbuffers in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from onnxruntime-gpu==1.15.0->-r requirements.txt (line 17)) (23.5.26)
Requirement already satisfied: packaging in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from onnxruntime-gpu==1.15.0->-r requirements.txt (line 17)) (23.1)
Requirement already satisfied: colorama in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tqdm==4.65.0->-r requirements.txt (line 19)) (0.4.6)
Requirement already satisfied: addict in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.4.0)
Requirement already satisfied: future in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.18.3)
Requirement already satisfied: lmdb in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.4.1)
Requirement already satisfied: pyyaml in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (6.0)
Requirement already satisfied: tb-nightly in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.14.0a20230708)
Requirement already satisfied: yapf in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.40.1)
Requirement already satisfied: lpips in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.1.4)
Requirement already satisfied: gdown in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from codeformer-pip==0.0.4->-r requirements.txt (line 20)) (4.7.1)
Requirement already satisfied: basicsr>=1.4.2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from gfpgan==1.3.8->-r requirements.txt (line 21)) (1.4.2)
Requirement already satisfied: facexlib>=0.2.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from gfpgan==1.3.8->-r requirements.txt (line 21)) (0.3.0)
Requirement already satisfied: filterpy in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from facexlib>=0.2.5->gfpgan==1.3.8->-r requirements.txt (line 21)) (1.4.5)
Requirement already satisfied: numba in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from facexlib>=0.2.5->gfpgan==1.3.8->-r requirements.txt (line 21)) (0.57.1)
Requirement already satisfied: qudida>=0.0.4 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from albumentations->insightface==0.7.3->-r requirements.txt (line 6)) (0.0.4)
Requirement already satisfied: opencv-python-headless>=4.1.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from albumentations->insightface==0.7.3->-r requirements.txt (line 6)) (4.8.0.74)
Requirement already satisfied: imageio>=2.27 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-image->insightface==0.7.3->-r requirements.txt (line 6)) (2.31.1)
Requirement already satisfied: tifffile>=2022.8.12 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-image->insightface==0.7.3->-r requirements.txt (line 6)) (2023.7.4)
Requirement already satisfied: PyWavelets>=1.1.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-image->insightface==0.7.3->-r requirements.txt (line 6)) (1.4.1)
Requirement already satisfied: lazy_loader>=0.2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-image->insightface==0.7.3->-r requirements.txt (line 6)) (0.3)
Requirement already satisfied: humanfriendly>=9.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from coloredlogs->onnxruntime-gpu==1.15.0->-r requirements.txt (line 17)) (10.0)
Requirement already satisfied: six in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from gdown->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.16.0)
Requirement already satisfied: beautifulsoup4 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from gdown->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (4.12.2)
Requirement already satisfied: MarkupSafe>=2.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from jinja2->torch==2.0.1+cu118->-r requirements.txt (line 11)) (2.1.3)
Requirement already satisfied: contourpy>=1.0.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (1.1.0)
Requirement already satisfied: cycler>=0.10 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (0.11.0)
Requirement already satisfied: fonttools>=4.22.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (4.40.0)
Requirement already satisfied: kiwisolver>=1.0.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (1.4.4)
Requirement already satisfied: pyparsing<3.1,>=2.3.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (3.0.9)
Requirement already satisfied: python-dateutil>=2.7 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from matplotlib->insightface==0.7.3->-r requirements.txt (line 6)) (2.8.2)
Requirement already satisfied: wcwidth in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from prettytable->insightface==0.7.3->-r requirements.txt (line 6)) (0.2.6)
Requirement already satisfied: charset-normalizer<4,>=2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests->insightface==0.7.3->-r requirements.txt (line 6)) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests->insightface==0.7.3->-r requirements.txt (line 6)) (3.4)
Requirement already satisfied: urllib3<3,>=1.21.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests->insightface==0.7.3->-r requirements.txt (line 6)) (1.26.16)
Requirement already satisfied: certifi>=2017.4.17 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests->insightface==0.7.3->-r requirements.txt (line 6)) (2023.5.7)
Requirement already satisfied: joblib>=1.1.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-learn->insightface==0.7.3->-r requirements.txt (line 6)) (1.3.1)
Requirement already satisfied: threadpoolctl>=2.0.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from scikit-learn->insightface==0.7.3->-r requirements.txt (line 6)) (3.1.0)
Requirement already satisfied: mpmath>=0.19 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from sympy->torch==2.0.1+cu118->-r requirements.txt (line 11)) (1.3.0)
Requirement already satisfied: absl-py>=0.4 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.4.0)
Requirement already satisfied: grpcio>=1.48.2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.56.0)
Requirement already satisfied: google-auth<3,>=1.6.3 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.21.0)
Requirement already satisfied: google-auth-oauthlib<1.1,>=0.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.0.0)
Requirement already satisfied: markdown>=2.6.8 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (3.4.3)
Requirement already satisfied: setuptools>=41.0.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (67.8.0)
Requirement already satisfied: tensorboard-data-server<0.8.0,>=0.7.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.7.1)
Requirement already satisfied: werkzeug>=1.0.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.3.6)
Requirement already satisfied: wheel>=0.26 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.38.4)
Requirement already satisfied: importlib-metadata>=6.6.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from yapf->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (6.8.0)
Requirement already satisfied: platformdirs>=3.5.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from yapf->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (3.8.1)
Requirement already satisfied: tomli>=2.0.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from yapf->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.0.1)
Requirement already satisfied: cachetools<6.0,>=2.0.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from google-auth<3,>=1.6.3->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (5.3.1)
Requirement already satisfied: pyasn1-modules>=0.2.1 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from google-auth<3,>=1.6.3->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.3.0)
Requirement already satisfied: rsa<5,>=3.1.4 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from google-auth<3,>=1.6.3->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (4.9)
Requirement already satisfied: requests-oauthlib>=0.7.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from google-auth-oauthlib<1.1,>=0.5->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (1.3.1)
Requirement already satisfied: pyreadline3 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from humanfriendly>=9.1->coloredlogs->onnxruntime-gpu==1.15.0->-r requirements.txt (line 17)) (3.4.1)
Requirement already satisfied: zipp>=0.5 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from importlib-metadata>=6.6.0->yapf->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (3.15.0)
Requirement already satisfied: soupsieve>1.2 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from beautifulsoup4->gdown->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (2.4.1)
Requirement already satisfied: llvmlite<0.41,>=0.40.0dev0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from numba->facexlib>=0.2.5->gfpgan==1.3.8->-r requirements.txt (line 21)) (0.40.1)
Requirement already satisfied: PySocks!=1.5.7,>=1.5.6 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests->insightface==0.7.3->-r requirements.txt (line 6)) (1.7.1)
Requirement already satisfied: pyasn1<0.6.0,>=0.4.6 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (0.5.0)
Requirement already satisfied: oauthlib>=3.0.0 in f:\roop_unleashed_win\installer_files\env\lib\site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<1.1,>=0.5->tb-nightly->codeformer-pip==0.0.4->-r requirements.txt (line 20)) (3.2.2)
Launching App
Loading roop.processors.frame.face_swapper
Loading roop.processors.frame.face_enhancer
Frame processor face_enhancer not found.

Done!
Press any key to continue . . .

Still an error about the face enhancer not being found. So I press a key on my keyboard, the window disappears and nothing happens. I wait minutes and nothing happens. I wish I knew what was wrong because I've reinstalled roop like 4 times now and it's impossible to get it to work even though it was working yesterday.

**Details**
What OS are you using?
- [x] Windows 11

Are you try to use a GPU?
- [x] No. I am not using the `---gpu` flag

**Sanity Check**
- [x] I have the latest code from the github repository
- [x] I have followed the installation guide

**Comment by Zyberetro** [comment]
So I decided to try to restore the old version (1.1.0) by downloading 1.1.0's source code and running it using the .bat file. It failed to run without updating it, so I decided to try updating it. Somehow, updating it worked, a GUI now pops up. But the kicker is that it's the 1.1.0 version, not 2.0.0, and I'm frankly scared to update 1.1.0 again because I don't want to break my installation again (Don't fix what ain't broken, as they say). Though I'm surprised that when it updated and successfully loaded that it wasn't 2.0.0.
